scipy/special/tests/test_support_alternative_backends.py

```diff
@@ -47,20 +47,30 @@ def test_rel_entr_generic(dtype):
     xp_assert_close(res, xp.asarray(ref), xp=xp)
 
 
+class RNGKeeper:
+    # One random number stream per function tested
+    def __init__(self):
+        self.rngs = {}
+
+    def get_rng(self, f_name):
+        if f_name not in self.rngs:
+            self.rngs[f_name] = np.random.default_rng(234892348934265)
+        return self.rngs[f_name]
+
+
+rng_keeper = RNGKeeper()
+
+
 @pytest.mark.fail_slow(5)
 @array_api_compatible
 @given(data=strategies.data())
+# @pytest.mark.skip_xp_backends('numpy', reasons=['skip while debugging'])
+# @pytest.mark.usefixtures("skip_xp_backends")
```

Comment:
Intentionally left these here because I always forget the syntax and they are useful for debugging.